samples/CosmosDBSample/CosmosDBSample.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
In https://github.com/Azure/azure-webjobs-sdk-extensions/pull/448 I just migrated everything, so I didn't have to split any samples out.

Reply:
I actually liked the split-out samples -- it meant no more commenting out stuff before testing my extension. I was hoping to do it with all of them. But I don't care enough to fight for it -- so I'll get these non-sample changes merged with yours (so you get all the tests completed) and add it to your branch.